.changeset/six-birds-accept.md

```diff
@@ -0,0 +1,9 @@
+---
+"@bigcommerce/catalyst-core": minor
+---
+
+Added a default sample rate of 25% for both Vercel Speed Insights and Vercel Analytics. This provides a good jumping off point for merchants to tweak their sample rates if they choose to.
+
+In order to tweak the sample rate, you can add the following environment variables to your Vercel project:
+- `ANALYTICS_SAMPLE_RATE` for Vercel Analytics
+- `SPEED_INSIGHTS_SAMPLE_RATE` for Vercel Speed Insights
```

Comment:
Shouldn't these be prefixed with `NEXT_PUBLIC` if used within a client component?

Reply:
🤦‍♂️ yup

Reply:
🤦‍♂️ yup